src/event/event_config.h

```diff
@@ -133,7 +133,7 @@
 #	define EV_ONESHOT				0x0010
 #	define EV_CLEAR					0x0020
 #	define EV_DISPATCH				0x0080
-#	define EV_UDATA_SPECIFIC		0x0100
+#	define EV_UDATA_SPECIFIC			0x0000
```

Comment:
i.e.

```
#ifdef EV_UDATA_SPECIFIC
#	define DISPATCH_EV_DIRECT		(EV_UDATA_SPECIFIC|EV_DISPATCH)
#else
#	define DISPATCH_EV_DIRECT		0x0000
#	define EV_UDATA_SPECIFIC		0x0000
#	undef  EV_VANISHED
#	define EV_VANISHED				0x0000
#endif
```

that is there exactly for the cases where EV_UDATA_SPECIFIC isn't present (in both cases of backend, we also support the absence of this flag with kevent)